requirements/test.in

```diff
@@ -0,0 +1,5 @@
+coverage
+fixtures
+testtools
+hypothesis
+flake8
```

Comment:
> Hm, I initially thought that it would be fine to use python3 to run pip-tools, even if the project it was run against was Python 2. But maybe that's not true.

I dug into this a little bit. pip-tools *does* have a `--pip-args` that we could pass something like `--python-version 27` to, but it seems that those arguments don't get passed to the right part of pip (pip-tools uses pip internals to do the resolution, so defaults, like pip does, to getting dependencies for the running version of python). Thus, I think `pip-compile` needs to be run with the same python that is going to use the compiled requirements.